addon/lib/document.js

```diff
@@ -137,7 +137,10 @@ export default Base.extend({
     function() {
       if (this.parentDocument) return this.parentDocument.jexlContext;
 
-      return { form: this.rootForm.slug };
+      return {
+        form: this.rootForm.slug,
+        info: { root: { form: this.rootForm.slug } }
```

Comment:
We also have `info.parent.form`, which should also be implemented

Reply:
This is implemented in the field since it's not possible to get that context per document